source/index.html.md

```diff
@@ -808,6 +808,42 @@ object][ApplePay-PaymentToken] for more information.
   subsequent-in-series authorization.
 </p>
 
+Additionally, an Apple Pay authorization can be created using raw values from
+a payment token:
+
+<dl class="dl-vertical">
+  <dt>applepay[raw][pan]
+    <span class="type">[0-9]{12,19}</span>
+  </dt>
+  <dd>
+    Primary account number of card to charge.
+  </dd>
+  <dt>applepay[raw][expire_month]
+    <span class="type">[0-9]{2}</span>
+  </dt>
+  <dd>
+    Expiry month of card to charge.
+  </dd>
+  <dt>applepay[raw][expire_year]
+    <span class="type">20[0-9]{2}</span>
+  </dt>
+  <dd>
+    Expiry year of card to charge.
+  </dd>
+  <dt>applepay[raw][cryptogram]
+    <span class="type">[:base64:]{28}</span>
+  </dt>
+  <dd>
+    Online payment cryptogram.
+  </dd>
+  <dt>applepay[raw][eci]
+    <span class="type">[0-9]{1,2}</span>
+  </dt>
+  <dd>
+    Electronic Commerce Indicator.
```

Comment:
Maybe mention that it comes from `eciIndicator`?

Reply:
https://github.com/clearhaus/gateway-api-docs/pull/137/commits/1e6b4a3190907b1aff08633f01d01652e45b9002